Godeps/_workspace/src/k8s.io/kubernetes/pkg/securitycontextconstraints/provider.go

```diff
@@ -251,6 +251,14 @@ func (s *simpleProvider) ValidateContainerSecurityContext(pod *api.Pod, containe
 		}
 	}
 
+	if !s.scc.AllowEmptyDirVolumePlugin {
+		for _, v := range pod.Spec.Volumes {
+			if v.EmptyDir != nil {
+				allErrs = append(allErrs, fielderrors.NewFieldInvalid("VolumeMounts", v.Name, "EmptyDir Volumes are not allowed to be used"))
```

Comment:
I don't think this is the canonical format for a field error. Check out https://github.com/dgoodwin/origin/blob/block-emptydir-scc/Godeps/_workspace/src/k8s.io/kubernetes/pkg/securitycontextconstraints/provider.go#L264 for reference. I think you should address it as something like `volumes[index]` (and correctly prefixed).